scripts/addon/build.py

```diff
@@ -0,0 +1,361 @@
+#! /usr/bin/env python3
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import argparse
+import json
+import os
+from pathlib import Path
+import xml.etree.ElementTree as ET
+import tempfile
+import shutil
+import sys
+
+comment_types = {
+    "text": f"Standard text in a guide block",
```

Comment:
f-strings are not necessary when there's no variable in them (there are a few of these in this file)

```suggestion
    "text": "Standard text in a guide block",
```